capstone-frontend/src/components/CreateList.jsx

```diff
@@ -35,37 +35,60 @@ class CreateList extends Component {
       method: "GET",
     }).then(resp => resp.json());
 
-    document.location.href = `/listpage/${createdBookList[0].id}`;
+    this.setState({ loading: false, showModal: false, renderModal: false })
```

Comment:
`nit`: semicolon

Reply:
Ok, but I kind forget to put semicolons all over in Javascript.